examples/apptainer-rootful.yaml

```diff
@@ -19,21 +25,15 @@ containerd:
   system: false
   user: false
 provision:
-- mode: system
-  script: |
-    #!/bin/bash
-    set -eux -o pipefail
-    modprobe squashfs >/dev/null 2>&1 && exit 0
-    dnf install -y kernel-modules-$(uname -r)
```

Comment:
note: the ubuntu kernel already has squashfs support built-in, so no need to install extra kernel modules